src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const BASE_PRICE = days * 40;
+
+  if (days < SHORT_TERM) {
+    return BASE_PRICE;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return BASE_PRICE - SHORT_TERM_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
```

Comment:
Similar to the previous comment, you don't need to check `days >= LONG_TERM` here. If the code execution reaches this point, it's already guaranteed that `days` is at least `LONG_TERM`.